### ISSUE TITLE
Failed to locate Digital River ID on product page

### ISSUE THREAD

**Issue by Mezzerto** [philippnormann/nvidia-sniper#44]
Hello,

This morning (US) I noticed Sniper reporting that it failed to locate the Digital River ID on product page in the logs. Last night this seemed accessible. Should this be of concern?

Edit: watching 3080 - https://www.nvidia.com/en-us/geforce/graphics-cards/30-series/rtx-3080/

Thanks.

**Comment by randomslap** [comment]
I am getting this warning as well

**Comment by McDAlexander** [comment]
This is not an issue with the program, NVIDIA turned their API off and basically shut down their store. I imagine they're reworking their store again in preparation for the next drop of cards.

**Comment by Mezzerto** [comment]
> This is not an issue with the program, NVIDIA turned their API off and basically shut down their store. I imagine they're reworking their store again in preparation for the next drop of cards.

I agree this isn't an issue with the application per se. I just wanted to flag this as new behavior instead of the 504 error I typically see when the API is unavailable. That said, I only recently begun running Sniper, so this may indeed be normal.

**Comment by philippnormann** [comment]
@McDAlexander Is right :+1:  
This just means that the site is currently either overloaded or going through maintenance. Not an issue we can fix.

**Comment by dieplanes789** [comment]
> @McDAlexander Is right 👍
> This just means that the site is currently either overloaded or going through maintenance. Not an issue we can fix.

Damn any idea when you will have it set up for the new store set up? Not too concerned with the $3 I spent on the capcha system, but it would have been nice if this was mentioned in the notes in the main page that the software isnt working before I set it up and paid for the service.

**Comment by McDAlexander** [comment]
> > @McDAlexander Is right 👍
> > This just means that the site is currently either overloaded or going through maintenance. Not an issue we can fix.
> 
> Damn any idea when you will have it set up for the new store set up? Not too concerned with the $3 I spent on the capcha system, but it would have been nice if this was mentioned in the notes in the main page that the software isnt working before I set it up and paid for the service.

Nobody knows when the API will come back up or what format the store will be in at that point. Most likely it will contain some sort of Captcha in the checkout process, so your $3 isn't going to waste. I bet development on the tool won't really kick back off until the API comes back up from Nvidia's side in prep for the next drop of cards.

**Comment by dieplanes789** [comment]
Thank you, this drop has been an absolute nightmare with multiple points where I could have gotten a card had things been more clear (biggest one being not knowing I had to manually fill out a small form to instantly become an EVGA elite member even though I already qualified). I am running more tools and bots than I care to count just so I can get a single card.On Oct 5, 2020 3:43 PM, McDAlexander <notifications@github.com> wrote:





@McDAlexander Is right 👍

This just means that the site is currently either overloaded or going through maintenance. Not an issue we can fix.



Damn any idea when you will have it set up for the new store set up? Not too concerned with the $3 I spent on the capcha system, but it would have been nice if this was mentioned in the notes in the main page that the software isnt working before I set it up and paid for the service.



Nobody knows when the API will come back up or what format the store will be in at that point. Most likely it will contain some sort of Captcha in the checkout process, so your $3 isn't going to waste. I bet development on the tool won't really kick back off until the API comes back up from Nvidia's side in prep for the next drop of cards.
